circle.yml

```diff
@@ -7,15 +7,15 @@ machine:
     version: 2.7.9
 dependencies:
   pre:
-    - wget -q https://dl.bintray.com/sbt/debian/sbt-0.13.13.deb
-    - sudo dpkg -i sbt-0.13.13.deb
+    - wget -q https://dl.bintray.com/sbt/debian/sbt-1.0.2.deb
+    - sudo dpkg -i sbt-1.0.2.deb
   cache_directories:
     - "~/.ivy2"
     - "~/.sbt"
   override:
     - sbt +test:compile
 test:
   pre:
-    - sbt publishLocal
+    - sbt +publishLocal
   override:
-    - sbt test:test
+    - sbt +test:test
```

Comment:
You may notice that one part of our testing framework contains running the example project under the `example` dir. Right now, this part does not work correctly with cross testing, because the example project was always set to run on Scala 2.12... Fixing this is on my agenda but I haven't been able to work on it yet.